community/kernel/src/test/java/org/neo4j/kernel/impl/store/PropertyStoreTest.java

```diff
@@ -0,0 +1,129 @@
+package org.neo4j.kernel.impl.store;
```

Comment:
D'oh, sorry! Have put the header in now hopefully.

On 19 January 2015 at 15:47, Tobias Lindaaker <notifications@github.com>
wrote:

> In
> community/kernel/src/test/java/org/neo4j/kernel/impl/store/PropertyStoreTest.java
> <https://github.com/neo4j/neo4j/pull/3831#discussion-diff-23167248>:
>
> > @@ -0,0 +1,129 @@
> > +package org.neo4j.kernel.impl.store;
>
> Missing license header...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4j/neo4j/pull/3831/files#r23167248>.
>